src/redist/targets/GenerateRPMs.targets

```diff
@@ -177,27 +179,41 @@
     </PropertyGroup>
 
     <PropertyGroup>
-
       <SharedFxRpmPackageVersion>$(MicrosoftNETCoreAppMajorMinorVersion)</SharedFxRpmPackageVersion>
       <SharedFxRpmPackageFileVersion>$(MicrosoftNETCoreAppPackageVersion)</SharedFxRpmPackageFileVersion>
       <SharedFxRpmPackageName>dotnet-runtime-$(SharedFxRpmPackageVersion)</SharedFxRpmPackageName>
       <SharedFxRpmPackageName>$(SharedFxRpmPackageName.ToLower())</SharedFxRpmPackageName>
       <SharedFxRpmPackageFileName>dotnet-runtime-$(SharedFxRpmPackageFileVersion)</SharedFxRpmPackageFileName>
       <SharedFxRpmPackageFileName>$(SharedFxRpmPackageFileName.ToLower())</SharedFxRpmPackageFileName>
-      <HostFxrRpmPackageVersion>$(HostFxrVersion)</HostFxrRpmPackageVersion>
-      <HostFxrRpmPackageName>dotnet-hostfxr-$(HostFxrRpmPackageVersion)</HostFxrRpmPackageName>
+      <SharedFxTargetingPackRpmPackageName>dotnet-targeting-pack-$(SharedFxRpmPackageVersion)</SharedFxTargetingPackRpmPackageName>
```

Comment:
Name of the targeting pack is missing, this is the .NET Core targeting pack. Maybe match the MSI properties, call this `NetCoreAppTargetingPackRpmPackageName`.

(In general I suppose they *could* be called "shared framework targeting packs" but I can't remember seeing that done before.)

Reply:
Renamed...